doc/RELEASE_PROCESS.md

```diff
@@ -0,0 +1,25 @@
+# How to make a keyvi release (e.g., vX.Y.Z)
+
+## Create a release branch
+Create a release branch called `release-X.Y.Z`
+
+## Update the `setup.py` file
+ - Update the flag `IS_RELEASED` to `True`
+ - Commit to `release-X.Y.Z` and push it to https://github.com/KeyviDev/keyvi/
+ - Wait for travis to build all targets
+
+## Create tag 
+ - Draft a new release tagged vX.Y.Z with `release-X.Y.Z` as the target branch
+ - Add the release notes in the description with references to PRs
+ - Publish release
+
+## Update the `setup.py` file
+ - Update the flag `IS_RELEASED` to False
```

Comment:
I am not getting this part. As we are branching out for the release, all that needs to be done is bumping the version.